includes/class-wp-job-manager-post-types.php

```diff
@@ -73,6 +74,28 @@ public function __construct() {
 		$this->job_content_filter( true );
 	}
 
+	/**
+	 * Prepare CPTs for special block editor situations.
+	 */
+	public function prepare_block_editor() {
+		add_filter( 'allowed_block_types', array( $this, 'force_classic_block' ), 10, 2 );
+	}
+
+	/**
+	 * Forces job listings to just have the classic block. This is necessary with the use of the classic editor on
+	 * the frontend.
+	 *
+	 * @param array   $allowed_block_types
+	 * @param WP_Post $post
+	 * @return array
+	 */
+	public function force_classic_block( $allowed_block_types, $post ) {
+		if ( 'job_listing' === $post->post_type ) {
```

Comment:
I don't quite get why this is needed when the template is locked on the CPT. It seems to work just fine without the filter. Is there a "before" use case I can test to understand the impact of the filter?

Reply:
This is a bit of a future proof depending on how they fix the issue with converting classic block to blocks.

Reply:
I like this.